c/src/server.rs

```diff
@@ -464,6 +504,37 @@ pub extern "C" fn foxglove_server_broadcast_time(
     FoxgloveError::Ok
 }
 
+/// Publishes the current playback state to all clients.
+///
+/// Requires the `FOXGLOVE_CAPABILITY_RANGED_PLAYBACK` capability.
+///
+/// # Safety
+/// - `playback_state` must be a valid pointer to a playback state that lives for the duration of the call.
+#[unsafe(no_mangle)]
+pub unsafe extern "C" fn foxglove_server_broadcast_playback_state(
+    server: Option<&FoxgloveWebSocketServer>,
+    playback_state: Option<&FoxglovePlaybackState>,
+) -> FoxgloveError {
+    let Some(server) = server else {
+        return FoxgloveError::ValueError;
```

Comment:
Nit: Could be written more succinctly using map_err and ?

Reply:
I don't think we can use `?`, since this C-bound function returns an Error, not a Result.